.github/workflows/realease-images-from-main.yml

```diff
@@ -0,0 +1,42 @@
+name: Release named images
+# This action is meant to run on the main branch after a merge to release a named version of the image released
+# before with a hash as a tag.
+
+on:
+  push:
+    # ONLY main branch should be listed below!
+    branches: [ main ]
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron: '1 1 * * *'
+
+jobs:
+  retag:
+    name: Retag image & push
+    strategy:
+      matrix:
+        service_name: [ "sample-service", "practitioner-service", "trial-config-service" ]
```

Comment:
I think you can already add the role-service

Reply:
It is in the repo...

Reply:
done